monai/transforms/transform.py

```diff
@@ -136,8 +141,12 @@ def apply_transform(
         Union[List[ReturnType], ReturnType]: The return type of `transform` or a list thereof.
     """
     try:
-        if isinstance(data, (list, tuple)) and map_items:
-            return [_apply_transform(transform, item, unpack_items, lazy, overrides, log_stats) for item in data]
+        map_items_ = int(map_items) if isinstance(map_items, bool) else map_items
+        if isinstance(data, (list, tuple)) and map_items_ > 0:
+            return [
+                apply_transform(transform, item, map_items - 1, unpack_items, log_stats, lazy, overrides)
```

Comment:
```suggestion
                apply_transform(transform, item, map_items_ - 1, unpack_items, log_stats, lazy, overrides)
```
My bad, I should have copied the code from my test repo